extensions/wikia/Rail/scripts/Rail.js

```diff
@@ -37,6 +37,18 @@ $(function () {
 			params.excludeScss = window.wgSassLoadedScss;
 		}
 
+		// TODO XW-2760 remove after experiment is done
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)